# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLExtensionDrawBuffers.cpp
# Commit: ccfa767dba64
# Full Hash: ccfa767dba644dc193e0246eb8e8ff3377e8b8a5
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Client-side bindings mirror for precise CC, and merge similar codepaths. r=handyman
#   
#   * Context loss using RAII
#   * Move Program reflection Client-side
#   
# ==============================================================================

diff -r 71c122ac0ca7 -r ccfa767dba64 dom/canvas/WebGLExtensionDrawBuffers.cpp
--- a/dom/canvas/WebGLExtensionDrawBuffers.cpp	Wed Jan 08 22:19:14 2020 +0000
+++ b/dom/canvas/WebGLExtensionDrawBuffers.cpp	Wed Jan 08 22:19:16 2020 +0000
@@ -18,22 +18,6 @@
 WebGLExtensionDrawBuffers::WebGLExtensionDrawBuffers(WebGLContext* webgl)
     : WebGLExtensionBase(webgl) {
   MOZ_ASSERT(IsSupported(webgl), "Don't construct extension if unsupported.");
-
-  webgl->UpdateMaxDrawBuffers();
-}
-
-WebGLExtensionDrawBuffers::~WebGLExtensionDrawBuffers() {}
-
-void WebGLExtensionDrawBuffers::DrawBuffersWEBGL(
-    const nsTArray<GLenum>& buffers) {
-  if (mIsLost) {
-    if (mContext) {
-      mContext->ErrorInvalidOperation("drawBuffersWEBGL: Extension is lost.");
-    }
-    return;
-  }
-
-  mContext->DrawBuffers(buffers);
 }
 
 bool WebGLExtensionDrawBuffers::IsSupported(const WebGLContext* webgl) {